schedule/yam/agama_nvme.yaml

```diff
@@ -0,0 +1,16 @@
+---
+name: agama_nvme
+description: >
+  Prepare url for agama.auto boot parameter, perform auto-installation with nvme on agama.
+schedule:
+  - yam/agama/boot_agama
+  - yam/agama/agama_auto
+  - installation/grub_test
+  - installation/first_boot
+  - yam/validate/validate_product
+  - yam/validate/validate_first_user
```

Comment:
these two don't seem to be specific of nvme, please drop.

Reply:
removed.